app/views/publications/index.html.erb

```diff
@@ -9,9 +9,9 @@
 
 <header class="block headings-block">
   <div class="inner-block floated-children">
-    <%= render partial: 'shared/heading',
-              locals: { big: true,
-                        heading: heading } %>
+    <%= render "govuk_publishing_components/components/title", {
```

Comment:
Can we remove this page entirely? 

`https://www.gov.uk/government/publications.ka`

Reply:
I'm a little anxious to because:
1. I don't know for sure what the coverage of that pages is
2. I don't know how to safely remove routes in our apps 😬 

The latter I could probably find out. I'm pushing back on this because there was general agreement within the team that the work to remove potentially redundant templates in whitehall was a notch on the tech debt portion of our roadmap further down the line. Happy to chat if you've got an idea in mind that I'm missing.